compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeTransformer.java

```diff
@@ -5432,9 +5432,12 @@ private BLangMarkdownDocumentation createMarkdownDocumentationAttachment(Optiona
 
                     BLangIdentifier paraName = new BLangIdentifier();
                     Token parameterName = parameterDocLineNode.parameterName();
-                    paraName.value = parameterName.isMissing() ? "" : parameterName.text();
+                    var parameterNameValue = parameterName.isMissing() ? "" : parameterName.text();
                     bLangParaDoc.parameterName = paraName;
```

Comment:
The order is a bit messy. We assign the `paraName` to `bLangParaDoc.parameterName` and then modify the `paraName` again. Shall we move the assignment to the end?